js/private/js_binary.bzl

```diff
@@ -181,6 +148,24 @@ _ATTRS = {
         which can lead to non-hermetic behavior.""",
         default = True,
     ),
+    "include_transitive_sources": attr.bool(
+        doc = """When True, 'transitive_sources' from 'JsInfo' providers in data targets are included in the runfiles of the target.""",
+        default = True,
+    ),
+    "include_declarations": attr.bool(
+        doc = """When True, 'declarations' and 'transitive_declarations' from 'JsInfo' providers in data targets are included in the runfiles of the target.
+
+        Defaults to false since declarations are generally not needed at runtime and introducing them could slow down developer round trip
+        time due to having to generate typings on source file changes.""",
+        default = False,
+    ),
+    "include_npm_linked_packages": attr.bool(
+        doc = """When True, files in 'npm_linked_packages' and 'transitive_npm_linked_packages' from 'JsInfo' providers in data targets are included in the runfiles of the target.
```

Comment:
how come you used single quotes instead of backticks?

Reply:
I should fix vscode so you can just hit backtick and quote a selected word. Currently only works with single quote.